site/src/components/CreateUserForm/CreateUserForm.tsx

```diff
@@ -172,12 +171,41 @@ export const CreateUserForm: FC<
               }
               await form.setFieldValue("login_type", e.target.value)
             }}
+            SelectProps={{
+              renderValue: (selected: unknown) =>
+                authMethodLanguage[selected as keyof typeof authMethodLanguage]
+                  ?.displayName ?? "",
+            }}
           >
             {methods}
           </TextField>
+          <TextField
+            {...getFieldHelpers(
+              "password",
+              form.values.login_type === "password"
+                ? ""
+                : "No password required for this login type",
```

Comment:
If the password is not required, why not hide this field?

Reply:
jitter, mostly. on some level, seeing another field after the auth method prepares the user for "there is another potential step" even if it's disabled. if it's entirely hidden, they might just jump straight to the submit button.